lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/generic/JobDriver.java

```diff
@@ -134,9 +135,10 @@
    * Job driver constructor.
    * All parameters are injected from TANG automatically.
    *
-   * @param clock              Wake clock to schedule and check up running jobs.
-   * @param jobMessageObserver is used to send messages back to the client.
-   * @param evaluatorRequestor is used to request Evaluators.
+   * @param clock                      Wake clock to schedule and check up running jobs.
+   * @param jobMessageObserver         is used to send messages back to the client.
+   * @param evaluatorRequestor         is used to request Evaluators.
+   * @param activeContextBridgeFactory
```

Comment:
so the "singularity" of ActiveContextBridgeFactory is guaranteed by the Tang injection (with ActiveContextBridgeFactory ctor being private), right?